Source/WebCore/Modules/url-pattern/URLPatternCanonical.cpp

```diff
@@ -192,23 +192,13 @@ ExceptionOr<String> canonicalizeOpaquePathname(StringView value)
     if (value.isEmpty())
         return value.toString();
 
-    bool hasLeadingSlash = value[0] == '/';
-    // Prepend slash to disable URL parser from prepending slash.
-    // Prepend dash to avoid inadvertantly collapsing a leading dot due to the fake leading slash.
-    String maybeAddSlashPrefix = hasLeadingSlash ? value.toString() : makeString("/-"_s, value);
-
     // FIXME: Set state override to State::OpaquePath after URLParser supports state override.
```

Comment:
Maybe we can remove this FIXME as well? I don't think we really support state overrides currently and this implementation seems reasonably solid.

Reply:
Done